src/main/java/eu/bbmri_eric/negotiator/dto/syncjobservice/DiscoverySyncJobServiceCreateDTO.java

```diff
@@ -0,0 +1,18 @@
+package eu.bbmri_eric.negotiator.dto.syncjobservice;
+
+import jakarta.validation.constraints.NotNull;
+import lombok.AllArgsConstructor;
+import lombok.Builder;
+import lombok.Getter;
+import lombok.NoArgsConstructor;
+import lombok.Setter;
+
+@Getter
+@Setter
+@Builder
+@NoArgsConstructor
+@AllArgsConstructor
+public class DiscoverySyncJobServiceCreateDTO {
+
+  @NotNull private String discoveryServiceName;
```

Comment:
why are we still using Name not id?

Reply:
fixed in fb760484e9b2dbec0c0c5b584decfbd60305ad4d